enhancements/network/egress-qos.md

```diff
@@ -27,8 +29,9 @@ tracking-link:
 Not all traffic has the same priority, and when there is contention for bandwidth, there should be a mechanism for objects outside the cluster to prioritize the traffic.
 To enable this, we will use Differentiated Services Code Point (DSCP) which allows us to classify packets by setting a 6-bit field in the IP header, effectively marking the priority of a given packet relative to other packets as "Critical", "High Priority", "Best Effort" and so on.
 
-By introducing a new CRD `EgressQoS`, users could specify a DSCP value for packets originating from pods on a given namespace heading to a specified CIDR.
+By introducing a new CRD `QoS`, users could specify a DSCP value for packets originating from pods on a given namespace heading to a specified CIDR, Protocol and Port.
```

Comment:
Is the CRD name change implying we want to allow marking for ingress traffic? What are the use cases for this requirement because its very very rare that external entities talk to the pod directly by initiating connections right?
Mostly those connections happen via ingress controller or services. Looking forward to some real use cases for this requirement.

Reply:
good catch @tssurya, I also don't see a compelling use case for ingress pod traffic, but this may be useful for pod's secondary interfaces which run in a plain L2 network, but that's not managed by OVNK.